### PR TITLE
Add ITK filters (starting with median filter)

### PR DESCRIPTION
**Pull request by floryst**
Adds the median filter to Glance. You can play with it at https://paraview-glance-dev.netlify.com/ (Valid so long as this PR is active.)

@thewtex @aylward @HastingsGreer

### CODE REVIEW

**Comment by aylward** [comment]
Using the link:
https://paraview-glance-dev.netlify.com/
I could load the tooth, but I couldn't select or create an output image - so I could not run the filter.

**Comment by aylward** [comment]
Ah - I see I have to type a name into the "select or create an output image" box.  Then the filter will be enabled.    Perhaps say "Select or type a name to create an output image"

File: src/components/tools/MedianFilter/script.js [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+            type: 'error',
+            msg: 'Median filter error!',
+          });
+        });
+    },
+    showSnack({ type, msg }) {
+      this.snackbar.show = true;
+      this.snackbar.type = type;
+      this.snackbar.msg = msg;
+    },
```
Comment (thewtex) [inline src/components/tools/MedianFilter/script.js]: :yum:

**Comment by thewtex** [comment]
:heart:  !

image

**Pull request merged**